code/controllers/subsystem/ticker.dm

```diff
@@ -123,14 +123,14 @@ SUBSYSTEM_DEF(ticker)
 			current_state = GAME_STATE_FINISHED
 			Master.SetRunLevel(RUNLEVEL_POSTGAME) // This shouldnt process more than once, but you never know
 			auto_toggle_ooc(TRUE) // Turn it on
-
 			declare_completion()
+			addtimer(CALLBACK(src, .proc/call_reboot), 5 SECONDS)
 
-			spawn(50)
-				if(mode.station_was_nuked)
-					world.Reboot("Station destroyed by Nuclear Device.", "nuke")
-				else
-					world.Reboot("Round ended.", "proper completion")
+/datum/controller/subsystem/ticker/proc/call_reboot()
+	if(mode.station_was_nuked)
+		world.Reboot("Station destroyed by Nuclear Device.", "end_proper", "nuke")
+	else
+		world.Reboot("Round ended.", "end_proper", "proper completion")
```

Comment:
This has been conflicted with the SSblackbox PR. Please update and normalise back.

Reply:
What do you mean it has been conflicted?

Reply:
The args are now
```dm
/world/Reboot(var/reason, end_string, var/time)
```

When making this, you re-added the `"end proper"` string, which isnt valid anymore.

Reply:
Ah gotcha.